spec/intl.html

```diff
@@ -1588,7 +1594,7 @@ <h1>
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It takes two lists of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new list that include both sets without any redudant fields.</dd>
+            <dd>It takes two lists of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new list that include both sets without any redundant fields. This is relevant for calendars which accept fields other than the standard set of built-in calendar fields.</dd>
```

Comment:
> a new list that include both sets without any redundant fields

IMHO this isn't clear enough. I'd suggest the text below: (for some reason, GH's "suggestion" button isn't available):

> a new list that include both sets of fields. The values in _additionalFields_ should supersede the values in _fields_. Also, the returned field list must be free of ambiguity or conflicts. For example, if _fields_ contains _"year"_ but _additionalFields_ contains _"era"_ and _"eraYear"_, then the returned list must not include _"year"_ in order to avoid ambiguity or conflict between different ways of specifying the year.

Here's a suggested note to explain this AO: 

> This abstract operation supports calendars that allow a calendar unit to be provided in more than one way, e.g. where the year can be provided by a `year` field or a combination of `era` and `eraYear` fields.